core/arch/arm/pta/system.c

```diff
@@ -11,6 +12,17 @@
 
 #define MAX_ENTROPY_IN			32u
 
+/*
+ * Having keys with too few bits impose a potential security risk, hence set a
+ * lower bound (i.e., 128 bits).
+ */
+#define TA_DERIVED_KEY_MIN_SIZE 16
+
+/* Same value as max in huk_subkey_derive */
+#define TA_DERIVED_KEY_MAX_SIZE 32
+
+#define TA_DERIVED_EXTRA_DATA_MAX_SIZE 1024
```

Comment:
very minor: somewhat long but maybe prefix with `TA_DERIVED_KEY_` instead of `TA_DERIVED_`.